app/assets/stylesheets/common/base/sidebar.scss

```diff
@@ -105,285 +105,4 @@
-          white-space: nowrap;
-          overflow: hidden;
-          text-overflow: ellipsis;
-        }
-      }
-    }
-  }
-
-  .sidebar-section-message-wrapper {
-    display: flex;
-    margin-left: 1.5em;
-  }
-
-  .sidebar-section-message {
-    padding: 0.25em 0.5em;
-    font-size: var(--font-down-1);
-    color: var(--primary-high);
-  }
-
-  .sidebar-section-link-content-badge {
-    color: var(--tertiary);
-    font-size: var(--font-down-1);
-    font-weight: normal;
-    margin-left: auto;
-  }
-
-  .sidebar-section-header-caret {
-    flex: 0 0 auto;
-    width: 1.5em;
-    display: flex;
-    justify-content: flex-end;
-    align-items: center;
-    border: none;
-    background: transparent;
-    padding: 0;
-    opacity: 0;
-    transition: opacity 0.25s;
-    transition-delay: 0.5s;
-    .discourse-no-touch & {
-      &:hover {
-        svg {
-          color: var(--primary-high);
-        }
-      }
-    }
-
-    svg {
-      display: block;
-      flex: 0 0 auto;
-      transition: color 0.25s;
-    }
-
-    .d-icon {
-      font-size: $font-down-1;
-      color: var(--primary-medium);
-    }
-  }
-
-  .sidebar-section-content {
-    hr {
-      margin: 0em 1.5em;
-    }
-  }
-
-  .sidebar-section-link-personal-messages-sent,
-  .sidebar-section-link-personal-messages-new,
-  .sidebar-section-link-personal-messages-archive,
-  .sidebar-section-link-personal-messages-unread,
-  .sidebar-section-link-group-messages-new,
-  .sidebar-section-link-group-messages-unread,
-  .sidebar-section-link-group-messages-archive {
-    margin-left: 0.5em;
-  }
-}
-
-#main-outlet-wrapper .sidebar-section-wrapper {
-  .sidebar-section-link-prefix {
-    &.image {
-      img {
-        border-radius: 50%;
-        width: 20px;
-        aspect-ratio: auto 20 / 20;
-        height: 20px;
-        margin-right: 0.75em;
-      }
-      &.active img {
-        box-shadow: 0px 0px 0px 1px var(--success);
-        border: 1px solid var(--secondary);
-      }
-    }
-    &.text {
-      display: flex;
-      border-radius: 3px;
-      background: rgba(var(--primary-rgb), 0.1);
-      text-align: center;
-      font-weight: 700;
-      font-size: var(--font-down-1);
-      align-items: center;
-      padding: 0.25rem 0.5rem;
-      margin-right: 0.75em;
-    }
-    &.icon {
-      position: relative;
-      margin-right: 0.75em;
-      svg.prefix-badge {
-        position: absolute;
-        background-color: var(--secondary);
-        border-radius: 50%;
-        padding: 2px 2px 3px;
-        color: var(--primary-high);
-        height: 0.5rem;
-        width: 0.5rem;
-        margin-left: -0.4rem;
-      }
-    }
-  }
-  .sidebar-section-link-suffix.icon {
-    align-items: center;
-    display: flex;
-    margin-left: 0.5em;
-    svg {
-      width: 0.75em;
-      height: 0.75em;
-    }
-    &.urgent svg {
-      color: $success;
-    }
-    &.unread svg {
-      color: var(--tertiary-med-or-tertiary);
-    }
-  }
-  &.sidebar-section-chat-dms {
```

Comment:
@lis2 Minor but do note that we do not want to be including chat specific stuff in core.